[debug] [2019-07-20T06:26:02.950Z] ----------------------------------------------------------------------
[debug] [2019-07-20T06:26:02.956Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\MARITIM\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-20T06:26:02.956Z] CLI Version:   7.1.1
[debug] [2019-07-20T06:26:02.957Z] Platform:      win32
[debug] [2019-07-20T06:26:02.957Z] Node Version:  v10.15.3
[debug] [2019-07-20T06:26:02.961Z] Time:          Sat Jul 20 2019 09:26:02 GMT+0300 (East Africa Time)
[debug] [2019-07-20T06:26:02.961Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=890290151&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-20T06:26:06.941Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-20T07:10:28.790Z] ----------------------------------------------------------------------
[debug] [2019-07-20T07:10:28.881Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\MARITIM\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-20T07:10:28.881Z] CLI Version:   7.1.1
[debug] [2019-07-20T07:10:28.882Z] Platform:      win32
[debug] [2019-07-20T07:10:28.882Z] Node Version:  v10.15.3
[debug] [2019-07-20T07:10:30.026Z] Time:          Sat Jul 20 2019 10:10:28 GMT+0300 (East Africa Time)
[debug] [2019-07-20T07:10:30.027Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=462036405&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-20T07:11:41.174Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
